### PR TITLE
Bump @nextcloud/vue to v7.11.1

### PR DESCRIPTION
**Pull request by artonge**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "@nextcloud/paths": "^2.1.0",
     "@nextcloud/router": "^2.0.0",
     "@nextcloud/sharing": "^0.1.0",
-    "@nextcloud/vue": "^7.10.0",
+    "@nextcloud/vue": "^7.11.1",
```
Comment (szaimen) [inline package.json:56]: The heading does not fit

Comment (artonge) [reply package.json:56]: True, sneaky release.
Fixed :)

**Review by szaimen — APPROVED** [review]

**Review by susnux — APPROVED** [review]

**Comment by susnux** [comment]
The sidebar tabs changed, so at least the acceptance tests need to be adjusted to find the tab toggles:
https://github.com/nextcloud/server/blob/1a00d99015d352d3856d0a8c2748abd1b709156a/tests/acceptance/features/bootstrap/FilesAppContext.php#L199

**Review by JuliaKirschenheuter — APPROVED** [review]

**Comment by susnux** [comment]
Superseded by https://github.com/nextcloud/server/pull/38124

**Pull request closed without merging**